src/test/Figure/Population.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class PopulationResponse(CamelCaseModel):
+    label: str = Field(..., title="Data reference label")
```

Comment:
If I'd be building a productizer for this definition I'd not know what to enter here. Maybe some sort of longer summary/description could be useful. I'm also wondering (depending on what it is for) if it should possibly be optional and default to None.